### PR TITLE
Add MCP and Config Version columns to MCN output

### PR DESCRIPTION
**Pull request by dkhater-redhat**
Updated MCN output to include the pool name and config versions. Changed these to a higher priority and redid the priorities on the other columns so they only show up in `-o wide`

### CODE REVIEW

File: machineconfiguration/v1alpha1/types_machineconfignode.go [hunk 1]
```diff
@@ -13,6 +13,9 @@ import (
 // +openshift:api-approved.openshift.io=https://github.com/openshift/api/pull/1596
 // +openshift:file-pattern=cvoRunLevel=0000_80,operatorName=machine-config,operatorOrdering=01
 // +openshift:enable:FeatureGate=MachineConfigNodes
+// +kubebuilder:printcolumn:name="MCP",type="string",JSONPath=.spec.pool.name,priority=1
```
Comment (JoelSpeed) [inline machineconfiguration/v1alpha1/types_machineconfignode.go:16]: Should this just be Pool Name? Not sure what others do but initialisms here seem a touch odd to me. Let's check a few other APIs for examples

Comment (dkhater-redhat) [reply machineconfiguration/v1alpha1/types_machineconfignode.go]: i dont mind chaning to `PoolName` 

when you say "initialisms" do you mean the jsonpath? i ran this and it pulled the specs just fine.

**Comment by JoelSpeed** [comment]
Changes look good, have you tested what the output of an `oc get machineconfignodes` and `oc get machineconfignodes -o wide` looks like here?

**Comment by dkhater-redhat** [comment]
Had a conversation with Joel and moved all of the required output columns to priority 0. moved the rest to priority 1 so its available when specifying `-o wide`

**Pull request merged**